sidebars.js

```diff
@@ -139,6 +139,13 @@ module.exports = {
       "api/visualizer/read",
       "api/visualizer/update",
       "api/visualizer/delete"
+    ],
+    "Contacts group": [
+      "api/contacts-group/overview",
+      "api/contacts-group/create",
+      "api/contacts-group/read",
+      "api/contacts-group/update",
+      "api/contacts-group/delete"
     ]
```

Comment:
Lets move this sidebar navigation higher up so that it falls within the correct alphabetical range